LICENSE-MIT

```diff
@@ -0,0 +1,22 @@
+Copyright (c) 2018 
```

Comment:
What was the reason to use the MIT license?
We generally use the GPL 2+ license for all our projects.